src/provider/bn256_grumpkin.rs

```diff
@@ -8,7 +8,7 @@ use digest::{ExtendableOutput, Update};
 use ff::{FromUniformBytes, PrimeField};
 use group::{cofactor::CofactorCurveAffine, Curve, Group as AnotherGroup};
 #[cfg(any(target_arch = "x86_64", target_arch = "aarch64"))]
-use grumpkin_msm::{bn256 as bn256_msm, grumpkin as grumpkin_msm};
+use grumpkin_msm::grumpkin as grumpkin_msm;
```

Comment:
Why is it that the Grumpkin msm should still be on sppark? Doesn't ingonyama support grumpkin as well?